sturdy/validator/forward.py

```diff
@@ -126,6 +127,8 @@ async def query_and_score_miners(
     # TODO: write custom availability function later down the road
     active_uids = [str(uid) for uid in range(self.metagraph.n.item()) if self.metagraph.axons[uid].is_serving]
 
+    random.shuffle(active_uids)
```

Comment:
i think it's better to just use `np.random.shuffle(active_uids)` here. one less import. i'll make this change now.